src/mappers/mapLiteral.ts

```diff
@@ -39,7 +39,7 @@ export default function mapLiteral(context: ParseContext, node: Literal): Node {
     return new JavaScript(line, column, start, end, raw, virtual, node.value);
   }
 
-  if (startToken.type === SourceType.NUMBER) {
+  if (startToken.type === SourceType.NUMBER || startToken.type === SourceType.DOT) {
```

Comment:
Oh, good point. Yeah, I guess probably the ideal solution is for coffee-lex to be smarter here and just call the whole thing a number.